heureka/ui/src/lib/slices/createGlobalsSlice.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { produce } from "immer"
+import constants from "../../components/shared/constants"
+
+const createGlobalsSlice = (set, get, options) => ({
+  globals: {
+    embedded: false,
```

Comment:
all of these state values need a comment to explain what they are. Not all are self-explanatory.